tests/test_slash_list.py

```diff
@@ -180,5 +180,42 @@ def test_slash_list_with_filtering(tmpdir):
     assert listed_tests == ['{}:{}'.format(fp.name, test_name) for test_name in ('test_a', 'test_c')]
 
 
+def test_slash_list_with_parameterization_filtering(tmpdir):
+    from textwrap import dedent
```

Comment:
I just noticed that `textwrap` is not an official dependency, which means it could break in the future... It's not a problem with this PR, but we should fix it probably...

Reply:
isn't it part of the standard lib?

Reply:
Doh! I had it confused with `texttable`. Yes it is.